src/main/java/org/jenkinsci/plugins/sonargerrit/gerrit/GerritAuthenticationConfig.java

```diff
@@ -123,10 +127,14 @@ public ListBoxModel doFillHttpCredentialsIdItems(
     }
 
     @SuppressWarnings(value = "unused")
+    @RequirePOST
     public FormValidation doTestConnection(
         @AncestorInPath Item item,
         @QueryParameter("httpCredentialsId") final String httpCredentialsId,
-        @QueryParameter("serverName") final String serverName) {
+        @QueryParameter("serverName") final String serverName)
+        throws ServletException, IOException {
+      checkPermission(item, Item.CONFIGURE);
```

Comment:
Should be `item.checkPermission(Item.CONFIGURE)`; `Functions` exists basically to support Jelly views.

Reply:
Thank you.
Will be fixed in https://github.com/jenkinsci/sonar-gerrit-plugin/pull/142 .